tests/phpunit/CRM/Member/Form/MembershipTest.php

```diff
@@ -1419,38 +1419,6 @@ protected function createTwoMembershipsViaPriceSetInBackEnd(int $contactId, $isT
     $form->testSubmit($params);
   }
 
-  /**
-   * Test membership status overrides when contribution is cancelled.
-   *
-   * @throws \CRM_Core_Exception
-   */
-  public function testContributionFormStatusUpdate(): void {
-    // @todo figure out why financial validation fails with this test.
-    $this->isValidateFinancialsOnPostAssert = FALSE;
-    $this->_contactID = $this->ids['Contact']['order'] = $this->createLoggedInUser();
-    $this->createContributionAndMembershipOrder();
-
-    $params = [
-      'total_amount' => 50,
-      'financial_type_id' => 2,
-      'contact_id' => $this->_contactID,
-      'payment_instrument_id' => CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'payment_instrument_id', 'Check'),
-      'contribution_status_id' => CRM_Core_PseudoConstant::getKey('CRM_Contribute_BAO_Contribution', 'contribution_status_id', 'Cancelled'),
-    ];
-    $_REQUEST['action'] = 'update';
-    $_REQUEST['id'] = $this->ids['Contribution'][0];
-    $form = $this->getContributionForm($params);
-    $form->postProcess();
-    $membership = $this->callAPISuccessGetSingle('Membership', ['contact_id' => $this->_contactID]);
-
-    //Assert membership status overrides when the contribution cancelled.
-    $this->assertEquals(TRUE, $membership['is_override']);
```

Comment:
@eileenmcnaughton this seemed to be the only assertion that this test was providing that the extensions ' test CancelFromContributionForm wasn't